### ISSUE TITLE
Support iOS 15

### ISSUE THREAD

**Issue by dnet890** [lavalleeale/Lemmios#67]
Hi
not everyone can support iOS 16 and it will be great that this app support iOS 15. What do you think?

**Comment by lavalleeale** [comment]
Yeah I would love that but it seems like there are far too many features this app uses that require at least iOS 16 that makes the port more work than I am willing to take on right now. If anyone else wants I would love the accept a pull request however
